### DISCUSSION TITLE
Filesystem & SSH

### DISCUSSION THREAD

**Discussion by AlexPadd** [GitSquared/edex-ui#896] [Ideas]
**Perceived idea awesomeness**
- [x] Wow, you don't have this yet???
- [x] Would be super cool.
- [ ] I'm not sure a lot of people would use it, but I'd love to have it.
- [ ] Kind of a gadget, honestly, but why not?

**Perceived idea difficulty**
- [ ] Quit your job first.
- [ ] Would need some work, but nothing impossible.
- [x] Shouldn't be too hard to make.
- [ ] Like, *I* could do it.

---

filesystem tab doesn't work in ssh
It doesn't show files in remote dir, but clicks on folders (or files) still work

Would be cool if FS tab will work in ssh too

**Comment by Animus-Surge** [comment]
While searching through the host filesystem is pretty easy to do, remotely doing this through SSH is another story. Unfortunately, and someone who knows their way around SSH much better than I do can correct me if I'm wrong, SSH doesn't have the ability to scan filesystems. (Yes there's the LS or DIR command, but that's in the actual shell, not the filesystem). Maybe if there could be an application on both machines to communicate the filesystem, but that's also a lot of work, and isn't very secure.

**Comment by GitSquared** [comment]
> Maybe if there could be an application on both machines to communicate the filesystem, but that's also a lot of work, and isn't very secure.

[Sounds about right](https://github.com/GitSquared/edex-ui-server). I never finished this, and it was the feature I was waiting for before releasing v3. :/
